bin/install

```diff
@@ -57,7 +60,11 @@ function install_rabbitmq {
 	# If rabbitmq $version >= 3.6.0
 	if [[ ${_semver//.} -ge '360' ]]; then
 		url=https://github.com/rabbitmq/rabbitmq-server/releases/download/$version/$os_prefix-$_semver.tar.xz
-		curl -L $url --output $tmp_download_dir/rabbitmq.tar.xz
+		status=$(curl --write-out %{http_code} -L $url --output $tmp_download_dir/rabbitmq.tar.xz)
+		if [ $status -ne 200 ]; then
```

Comment:
```suggestion
		if [ "$status" -ne 200 ]; then
```

this would fail in shellcheck but that's clearly the case for everything in this file, I'll make a separate issue for that